apps/src/code-studio/components/progress/teacherPanel/SelectedStudentInfo.jsx

```diff
@@ -117,8 +129,7 @@ export default class SelectedStudentInfo extends React.Component {
           {paired && (
             <div>
               <div>{i18n.workedWith()}</div>
-              {isDriver && <div>{i18n.partner({partner: partner})}</div>}
-              {isNavigator && <div>{i18n.loggedIn({partner: partner})}</div>}
+              {this.renderPartners(levelWithProgress)}
```

Comment:
Are we no longer distinguishing who the driver was?

Reply:
Correct, there will be no way to tell in the UI which student was the driver and which student(s) were the navigator(s).